cmd/kwil-cli/cmds/root.go

```diff
@@ -16,7 +16,39 @@ import (
 	"github.com/kwilteam/kwil-db/cmd/kwil-cli/config"
 )
 
+var longDesc = `Command line interface client for using %s.
+	
+	The %s CLI is a command line interface for interacting with %s.  It can be used to deploy, update, and query databases.
+	
+	The %s CLI can be configured with a persistent configuration file.  This file can be configured with the '%s configure' command.  The %s CLI will look for a configuration file at ` + "`" + `$HOME/.kwil-cli/config.json` + "`" + `.`
+
 func NewRootCmd() *cobra.Command {
+	return CustomRootCmd("kwil-cli", "Kwil")
+}
+
+func CustomRootCmd(usage, projectName string) *cobra.Command {
+	rootCmd := &cobra.Command{
+		Use:               usage,
+		Short:             fmt.Sprintf("Command line interface client for using %s.", projectName),
+		Long:              fmt.Sprintf(longDesc, projectName, projectName, projectName, projectName, usage, projectName),
```

Comment:
It seems the 3rd,5th,6th,7th should use `usage`?

Reply:
rn, the output is :
```
 ./Y client 
Command line interface client for using Yaiba.
        
        The Yaiba CLI is a command line interface for interacting with Yaiba.  It can be used to deploy, update, and query databases.
        
        The Yaiba CLI can be configured with a persistent configuration file.  This file can be configured with the 'client configure' command.  The Yaiba CLI will look for a configuration file at `$HOME/.kwil-cli/config.json`.
```

after changes:
```
 ./yaiba client 
Command line interface client for using Yaiba.
        
        client is a command line interface for interacting with Yaiba.  It can be used to deploy, update, and query databases.
        
        client can be configured with a persistent configuration file.  This file can be configured with the 'client configure' command.  The client will look for a configuration file at `$HOME/.kwil-cli/config.json`.
```

hmm, a bit weird..

Reply:
Nope. See how I use this when making `kwild`

Reply:
We're using `Y client`, and the help message is `Yaiba CLI`, shouldn't it be `client`?

Reply:
```
Command line interface client for using Kwil.

The Kwil CLI is a command line interface for interacting with Kwil.  It can be used to deploy, update, and query databases.

The Kwil CLI can be configured with a persistent configuration file.  This file can be configured with the 'client configure' command.  The Kwil CLI will look for a configuration file at `$HOME/.kwil-cli/config.json`.
```